### PR TITLE
Add kas and fix race in pipeline

### PR DESCRIPTION
**Pull request by mlilien**

### CODE REVIEW

**Review by JanZachmann — APPROVED** [review]

File: yocto-builder/Dockerfile [hunk 1]
```diff
@@ -22,7 +23,9 @@ RUN apt-get update \
       wget \
       xterm \
     && apt-get autoremove \
-    && apt-get clean
+    && apt-get clean \
+    && rm -rf /var/lib/apt/lists/*
+RUN pip3 install kas
```
Comment (JoergZeidler) [inline yocto-builder/Dockerfile:28]: is that tool used by anyone in yocto-builder? should be some information available in the readme?

Comment (mlilien) [reply yocto-builder/Dockerfile:28]: What do you expect exactly?

Comment (JoergZeidler) [reply yocto-builder/Dockerfile:28]: I understood you to say that the kas tool still needs to be configured. Or can it be that this PR is only used to install the tool?

Comment (mlilien) [reply yocto-builder/Dockerfile:28]: `kas` is only added in this PR... 
The configuration of a yocto image is project specific. We could add an example how to use kas in context of bb-yocto-builder but this is not refined. 
For me using `kas` here, e.g. in `pipeline.yaml` is another, not written yet, user story. Would we describe the current bb-yocto-builder way and an alternative way via `kas`? Or will `kas` replace the current implementation? Currently we didn't really verify `kas`. 

@JanZachmann Your opinion?

Comment (mlilien) [reply yocto-builder/Dockerfile:28]: Changed my mind. I'll use `kas` in `bb-yocto-builder` as well.

Comment (mlilien) [reply yocto-builder/Dockerfile]: done

**Review by JanZachmann — APPROVED** [review]

**Review by JoergZeidler — APPROVED** [review]

**Pull request merged**